dev/parm/config/gfs/config.resources

```diff
@@ -573,30 +573,36 @@ case ${step} in
     ;;
 
   "marineanlecen")
-    ntasks=16
     walltime="00:10:00"
+    # Decide how many ensemble members to process per MPI task (has to be divisor of NMEM_ENS).
+    if (( NMEM_ENS >= 10 && NMEM_ENS % 5 == 0 )); then
+      export NMEM_ENS_PER_MPI=5
+    elif (( NMEM_ENS % 2 == 0 )); then
+      export NMEM_ENS_PER_MPI=2
+    else
+      export NMEM_ENS_PER_MPI=1
+    fi
     case ${OCNRES} in
       "025")
-        ntasks=256
-        if [[ "${NMEM_ENS}" -gt 30 ]]; then
-          memory="500GB"
-          walltime="01:30:00"
-        else
-          ntasks=40
-          memory="256GB"
-        fi
+        ntasks=$(( 12 * NMEM_ENS / NMEM_ENS_PER_MPI ))
+        memory="${mem_node_max}"
+        walltime="00:40:00"
```

Comment:
@shlyaeva , this needs to be 30mn or less (hopefully much less). Anything above would require a restart feature.